bot/settings.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+from dotenv import load_dotenv
+load_dotenv()
+
+token = os.getenv('TG_TOKEN')
```

Comment:
can we replace env variable with command-line argument? I hate env vars

Reply:
and you should hate them too

Reply:
inho cli args for configuration (params), but secrets should be in files (`.env` files)